test/unit/test_scenario.py

```diff
@@ -101,7 +102,11 @@ def test_dependency_sequence_property(_instance):
 
 
 def test_destroy_sequence_property(_instance):
-    assert ['destroy'] == _instance.destroy_sequence
+    x = [
```

Comment:
I know this `x = ...` is everywhere in the code base but we're trying to get away from it. Could you rename? Thanks!

Reply:
I'd rather not create an inconsistency like that, and I'd also rather not clutter up this PR with a second commit making the change file-wide. My preferred approach would be to leave it like this and open up a second PR that fixes the `x = ...` problem for the entire file in one atomic action.

Reply:
You can ammend your commit? As you want though!

Reply:
Given the relative ease of fixing terse identifiers in the tests versus this way-more-involved PR, I think I'll just go clean up test_scenario.py real quick.

Reply:
PR posted: https://github.com/ansible/molecule/pull/1760